src/libtriton/engines/symbolic/symbolicEngine.cpp

```diff
@@ -963,6 +963,7 @@ namespace triton {
         SharedSymbolicExpression se         = nullptr;
         triton::uint64 address              = mem.getAddress();
         triton::uint32 writeSize            = mem.getSize();
+        auto id                             = this->uniqueSymExprId;
```

Comment:
Can we avoid as much as possible the `auto` keyword? I think it's more readable for ppl who want to take a look at Triton internals than reading the prototype of what `uniqueSymExprId` is. For those kind of declaration I used to use the type `triton::usize`.

However, I can accept the `auto` keyword when lines are too long without it or in loops like: `for (auto& e : l)`. Semantics files are the only exception.

Reply:
Ok, I'll fix it and try to remember this for future.